app/views/users/index.html.erb

```diff
@@ -1 +1,4 @@
-<h1> Users list </h1>
\ No newline at end of file
+<h1> Users list </h1>
+<% @users.each do |user| %>
+  <%= render partial: "shared/usercard", locals: {user: user}%>
+<% end %>
```

Comment:
- [ ] Please edit the user index view design to be like the given design and kindly ensure that you add all elements in the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/180582924-1fbdf925-f772-4849-b7eb-934abe38d7d4.png)  | ![image](https://user-images.githubusercontent.com/68796232/180582950-9ff70740-a261-4c42-80d3-4fcf94007382.png)  |